src/grants.ts

```diff
@@ -1,10 +1,27 @@
 import { JWTPayload } from 'jose';
+import { TrackSource } from './proto/livekit_models_pb';
+
+export function trackSourceToString(source: TrackSource) {
+  switch (source) {
+    case TrackSource.CAMERA:
+      return 'camera';
+    case TrackSource.MICROPHONE:
+      return 'microphone';
+    case TrackSource.SCREEN_SHARE:
+      return 'screen_share';
+    case TrackSource.SCREEN_SHARE_AUDIO:
+      return 'screen_share_audio';
+    default:
+      throw new TypeError(`Cannot convert TrackSource ${source} to string`);
+  }
+}
 
-export enum TrackSource {
-  CAMERA = 'camera',
-  MICROPHONE = 'microphone',
-  SCREEN_SHARE = 'screen_share',
-  SCREEN_SHARE_AUDIO = 'screen_share_audio',
+export function claimsToJwtPayload(grant: ClaimGrants): JWTPayload {
+  const claim: Record<string, unknown> = { ...grant };
```

Comment:
isn't grant a `ClaimGrants`? so we'd need to check under `grant.video`?

would be great to have a unit test for this helper to ensure proper behavior.

Reply:
yeah, thanks for catching that. Added a unit test, too